packages/extension/src/ui/features/accountTokens/tokens.state.ts

```diff
@@ -136,7 +136,7 @@ export const useNonFungibleTokensWithBalance = (
       const nonFungibleTokens: BaseTokenWithBalance[] = []
       for (const token of potentialNonFungibleTokens) {
         if (nonFungibleTokens.findIndex((t) => t.id == token.id) === -1) {
-          const tokenType = await fetchImmutable(`${token.id}-token-type`, () => nodeProvider.guessStdTokenType(token.id))
+          const tokenType = await fetchImmutable(`${token.id}-token-type`, () => nodeProvider.guessStdInterfaceId(token.id))
```

Comment:
With `1.2.2`, can we still use `guessStdTokenType`?

Reply:
Yes we can, we should use `guessStdTokenType` here.